quantum/nodes/circuit-diagram/circuit-diagram.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const snippets = require('../../snippets');
+const shell = require('../../python').PythonShell;
+
+module.exports = function(RED) {
+  function CircuitDiagramNode(config) {
+    RED.nodes.createNode(this, config);
+    this.name = config.name;
+
+    this.on('input', async function(msg, send, done) {
+      let script = snippets.CIRCUIT_BUFFER;
+
```

Comment:
This file should ensure all qubits have finished running before executing the Qiskit script (see 'simulator' node)

Right now, if the file has 2 inputs, he will generate twice the circuit diagram

Reply:
Thank you for your comment, I will modify it